site-src/concepts/use-cases.md

```diff
@@ -0,0 +1,177 @@
+    to Ana's application according to her routing specification.
+
+This allows Charlie to enforce centralized policies [such as
+TLS](/guides/tls#downstream-tls) at the Gateway, while simultaneously allowing
+Ana and her colleagues control over the application's [routing
+logic](/guides/http-routing) and rollout plans (e.g. [traffic splitting
+rollouts](/guides/traffic-splitting)).
+
+[north/south]:/concepts/glossary#north-south-traffic
+
+## Multiple applications behind a single Gateway
+
+This is remarkably similar to the [basic north/south use
+case](#basic-north-south-use-case), but there are multiple application teams:
+Ana and her team are managing a storefront application in the `store`
+Namespace, while Allison and her team are managing a website in the `site`
+Namespace.
+
+- Ian and Charlie work together to provide a cluster, `GatewayClass`, and
+  `Gateway`, as above.
+
+- Ana and Allison independently deploy workloads and HTTPRoutes bound to the
+  same `Gateway` resource.
+
+Again, this separation of concerns allows Charlie to enforce centralized
+policies [such as TLS](/guides/tls#downstream-tls) can be enforced at the
+Gateway. Meanwhile, Ana and Allison run their applications [in their own
+Namespaces](/guides/multiple-ns), but attach their Routes to the same shared
+Gateway, allowing them to independently control their [routing
+logic](/guides/http-routing), [traffic splitting
+rollout](/guides/traffic-splitting), etc., while not worrying about the things
+that Charlie and Ian are handling.
+
+[HTTPRoute]:/api-types/httproute
+[GatewayClass]:/api-types/gatewayclass
+[Gateway]:/api-types/gateway
+[Route Attachment Process]:/concepts/api-overview#attaching-routes-to-gateways
+
+![Gateway API Roles](/images/gateway-roles.png)
+
+## Basic [east/west] use case
+
+In this scenario, Ana has built a workload which is already running in a
+cluster with a [GAMMA]-compliant [service mesh]. She wants to use the mesh to
+protect her workload by rejecting calls to her workload with incorrect
+URL paths, and by enforcing timeouts whenever anyone makes a request of her
+workload.
+
+- Charlie and Ian have already provided a cluster with a running service mesh.
+  Ana doesn't need to make any requests of them.
+
+- Ana writes an HTTPRoute that defines acceptable routes and timeouts and has
+  a `parentRef` of her workload's Service.
+
+- Ana applies her HTTPRoute in the same Namespace as her workload.
+
+- The mesh automatically starts enforcing the routing policy described by
+  Ana's HTTPRoute.
+
+In this case, the separation of concerns across roles allows Ana to take
+advantage of the service mesh, with custom routing logic, without any
+bottlenecks in requests to Charlie or Ian.
+
+[east/west]:/concepts/glossary#east-west-traffic
+[GAMMA]:/contributing/gamma/
+[service mesh]:/concepts/glossary#service-mesh
+
+## Gateway and mesh use case
+
+This is effectively a combination of the `multiple applications behind a
```

Comment:
```suggestion
This is effectively a combination of the [multiple applications behind a
```

Was a broken link in deploy preview here: https://deploy-preview-2238--kubernetes-sigs-gateway-api.netlify.app/concepts/use-cases/